gradle/corset.gradle

```diff
@@ -16,16 +16,24 @@ tasks.register('corsetExists') {
     doLast {
         def result = exec{
             ignoreExitValue = true
-            commandLine "corset", "-V"
+            commandLine "go-corset", "--version"
```

Comment:
@letypequividelespoubelles We could decide to drop `CorsetExists` altogether.  But, for now, I just moved it over to `go-corset`.